features/machine/machine.feature

```diff
@@ -328,3 +328,37 @@ Feature: Machine features testing
       | attempting to acquire leader lease  openshift-machine-api/cluster-api-provider |
     """
 
+  # @author zhsun@redhat.com
+  # @case_id OCP-34718
+  @admin
+  @destructive
+  Scenario: Node labels and Affinity definition in PV should match	
+    Given I have an IPI deployment 
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+
+    # Create a pvc
+    Given I obtain test data file "cloud/pvc-34718.yml"
+    When I run the :create admin command with:
+      | f | pvc-34718.yml |
+    Then the step should succeed
+    And admin ensures "pvc-cloud" pvc is deleted after scenario
+
+    # Create a pod
+    Given I obtain test data file "cloud/pod-34718.yml"
+    When I run the :create admin command with:
+      | f | pod-34718.yml |
+    Then the step should succeed
+    And admin ensures "task-pv-pod" pod is deleted after scenario
```

Comment:
Create PVC/pod do not need admin privilege. Also we should not use project `openshift-machine-api` here as any project seems to be working here.

Reply:
We do not need line 336, 343 and 350. You can test that. If you met errors, replace client with admin in line 357, **or** switch to pseudo admin before line 354

Reply:
PVC/pod are project resources, they will be removed once the project is deleted, so no need explicitly clean-up (line 343 and 350). Also creating pvc/pod does not require storage-admin privilege (line 336).

Reply:
Thank you. Updated again, help to take a look.